### PR TITLE
#263 #269 #270 #272 #279 odni systems

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #263
 - #269 
 - #270 
 - #272 
 - #279

### CODE REVIEW

**Comment by ronaldtse** [comment]
Thank you @camobap!

I see that the systems only have minimal tests. Could you help make up some additional tests so that the systems can be well tested and remain stable even if someone else changes the rules? Thanks!

File: maps/odni-bel-Cyrl-Latn-2015.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+authority_id: odni
+id: 2015
+language: bel
+source_script: Cyrl
+destination_script: Latn
+name: Office of the Director Of National Intelligence Georgian Personal Names 2015, ICS 630-01 Annex B
```
Comment (antonsviridenko) [inline maps/odni-bel-Cyrl-Latn-2015.yaml]: "Georgian Personal Names" - it should be "Belarusian Personal Names" here. Same in other files

**Review by antonsviridenko — CHANGES_REQUESTED** [review]

File: maps/odni-bel-Cyrl-Latn-2015.yaml (hunk 1 above)
Comment (ronaldtse) [reply maps/odni-bel-Cyrl-Latn-2015.yaml]: Thank you @antonsviridenko you have a very sharp eye 😉

Comment (CAMOBAP) [reply maps/odni-bel-Cyrl-Latn-2015.yaml]: Done

**Comment by CAMOBAP** [comment]
More tests added

**Pull request merged**